pom.xml

```diff
@@ -115,6 +116,13 @@
                 <module>connectors/soap</module>
             </modules>
         </profile>
+        <profile>
+            <id>artemis</id>
```

Comment:
let's name this messaging-artemis.

In the beginning I thought of having separate profiles for separate areas eg: http, database, messaging and have packages or modules inside that profile and filter out relevant module when the tests are running. This is just to avoid the pom being cluttered with lots of profiles.
When doing that we need to make sure, when running tests for a given messaging connector, other connectors aren't getting unnecessarily built.

Anyway, for now let's keep this way and address it when adding a new messaging connector.

Reply:
Done